CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
```

Comment:
(So, GitHub review comments don't go away when the code they're commenting on is gone? Weird. That's part of what I loved about vanilla GitHub comments.)

Done.